plugin/plugin.go

```diff
@@ -107,3 +117,11 @@ var TimeBuckets = prometheus.ExponentialBuckets(0.00025, 2, 16) // from 0.25ms t
 
 // ErrOnce is returned when a plugin doesn't support multiple setups per server.
 var ErrOnce = errors.New("this plugin can only be used once per Server Block")
+
+type info string // info the key used to retrieve zone and addr info from the context.
```

Comment:
The context package advises to use pointers to struct types as keys to avoid any conflicts that could come from comparing plain values like strings.